lib/passenger.rb

```diff
@@ -18,7 +19,36 @@ def get_drivers
     end
 
     def add_trip(trip)
+      if trip.class != Trip
+        raise ArgumentError.new("Can only add trip instance to trip collection")
+      end
+
       @trips << trip
     end
+
+    def calculate_total_money_spent
+      if trips.length == 0
```

Comment:
Nice work checking different scenarios.  The 2nd gives a good argument for cost to return `nil` if the trip hasn't finished yet.